### PR TITLE
120 Update the execution table

### LINKED ISSUE #120: update the execution table to include submitter's username
step 3-4 need to have additional column to list submitter's username (at least) and more infos possible

Image

### PR DESCRIPTION
**Pull request by Rashmil-1999**
This PR modifies the execution info table to now indicate who ran the workflow and beautifies the date and time at which it was run.

### CODE REVIEW

**Review by nunezco2 — APPROVED** [review]

File: src/globals.js [hunk 1]
```diff
@@ -13,9 +13,9 @@ globals["maestroServiceBase"] = `${hostname}/maestro/galveston/`;
 globals["setGravatarURL"] = "https://en.gravatar.com/support/activating-your-account/";
 globals["allowUpload"] = true;
 globals["testbedSpace"] = "commresilienceslc";
-globals["testbedGroup"] = "incore_joplin_user"; // incore_galveston_user, incore_joplin_user, incore_slc_user
+globals["testbedGroup"] = "incore_slc_user"; // incore_galveston_user, incore_joplin_user, incore_slc_user
```
Comment (navarroc) [inline src/globals.js:16]: Is this change intentional? The updated table looks good, but I am wondering if this should be incore_joplin_user. @longshuicy ?

**Comment by navarroc** [comment]
Can you add a CHANGELOG entry?

File: src/globals.js (hunk 1 above)
Comment (longshuicy) [reply src/globals.js:16]: I think it should be ok. Since this is the "core" so it will never be really use independently, this is just for testing

**Review by longshuicy — APPROVED** [review]
